<launch>

    <node pkg="my_turtlebot3_vision" type="face_detector.py" name="face_detector" output="screen">

        <remap from="input_rgb_image" to="/image_raw" />

        <rosparam>
            haar_scaleFactor: 1.3
            haar_minNeighbors: 3
            haar_minSize: 30
            haar_maxSize: 150
        </rosparam>

        <param name="cascade_1" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_frontalface_alt2.xml" />
        <param name="cascade_2" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_frontalface_alt.xml" />
        <param name="cascade_3" value="$(find my_turtlebot3_vision)/data/haar_detectors/haarcascade_profileface.xml" />

    </node>

</launch>